internal/dev_server/sdk/go_sdk_test.go

```diff
@@ -123,4 +124,43 @@ func TestSDKRoutesViaGoSDK(t *testing.T) {
 			assert.Equal(t, value.AsArbitraryValue(), flagUpdate.NewValue.AsArbitraryValue())
 		})
 	}
+
+	newUpdates := map[string]ldvalue.Value{
+		"boolFlag":   ldvalue.Bool(true),
+		"stringFlag": ldvalue.String("fool"),
+		"intFlag":    ldvalue.Int(567),
+		"doubleFlag": ldvalue.Float64(99.99),
+		"jsonFlag":   ldvalue.CopyArbitraryValue(map[string]any{"tortoise": "hare"}),
+	}
+
+	// This test is testing the "put" payload in a roundabout way by verifying each of the flags are in there.
```

Comment:
```suggestion
	// This test is testing the "put" payload in a roundabout way by verifying each of the flags are in there.
	// Note that the store.UpsertOverride is called directly so that updates for the overrides are not sent via 
	// patches.
```